app/test/shared/configuration.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+import 'package:pub_dev/shared/configuration.dart';
+
+void main() {
+  test('Foo config from yaml file', () async {
```

Comment:
Let's also add another test:
- read the yaml file into `Configuration`
- serialize to JSON string, parse again
- serialize to JSON again and check if it is the same as the above.